tools/azure-sdk-tools/devtools_testutils/azure_testcase.py

```diff
@@ -268,9 +268,10 @@ def get_preparer_resource_name(self, prefix):
     @staticmethod
     def await_prepared_test(test_fn):
         """Synchronous wrapper for async test methods. Used to avoid making changes
-        upstream to AbstractPreparer (which doesn't await the functions it wraps)
+        upstream to AbstractPreparer, which only awaits async tests that use preparers.
+        (Add @AzureTestCase.await_prepared_test decorator to async tests without preparers)
         """
-
+# Note: This will only be needed so long as we maintain unittest.TestCase in our test-class inheritance chain.
```

Comment:
```suggestion
# Note: This function is only needed so long as we maintain unittest.TestCase in our test-class inheritance chain.
```
SUPER nitpicky suggestion only because if I were reading this as a glance I might take the unspecified comment as referring to the version_info conditional.